tests/acceptance/features/bootstrap/OcmContext.php

```diff
@@ -22,11 +22,14 @@
 use Behat\Behat\Context\Context;
 use Behat\Behat\Hook\Scope\BeforeScenarioScope;
 use GuzzleHttp\Exception\GuzzleException;
+use Psr\Http\Message\ResponseInterface;
 use TestHelpers\OcisHelper;
 use TestHelpers\OcmHelper;
 
+require_once 'bootstrap.php';
```

Comment:
is it required?

Reply:
no, I tried to set up settings.json 
I can remove it